splunk/help.md

```diff
@@ -220,27 +220,7 @@ This action is used to return the job history of a specified saved search.
 
 ### Triggers
 
-#### Alert
-
-This trigger allows you to configure a Splunk Alert to send data to Komand. It will add a webhook action to the Splunk actions to send the data to Komand.
-Any other webhook will be overwritten. You must have the name of the alert to hook.
-
-By default, this trigger makes a request every 15 seconds to establish the webhook.
-
-##### Input
-
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|interval|integer|15|False|Poll interval in seconds|None|
-|names|[]string|None|True|Names of the alerts to hook|None|
-
-##### Output
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|results_link|string|False|None|
-|result|object|False|None|
-|search_name|string|False|None|
+_This plugin does not contain any triggers._
```

Comment:
Alerts get removed?

Reply:
There is no trigger in plugin.spec.yaml

Reply:
Yea, this was removed in 3.0.0...it's Komand specific.